skipchain/skipchain.go

```diff
@@ -1348,16 +1318,11 @@ func (s *Service) bftForwardLinkAck(msg, data []byte) bool {
 // order that is more likely to give us non-failing subleaders in the byzcoinx
 // protocol.
 func (s *Service) startBFT(proto string, origRoster, newRoster *onet.Roster, msg, data []byte) (*byzcoinx.FinalSignature, error) {
+	// Before BDN signatures, the new roster was used when it was a rotation so
```

Comment:
This comment made sense to me until "but the workaround...", and then lost me. Can you improve that last part to be more clear?